src/main/java/com/ss/ita/rozetka/pageobject/pages/ProductTypePage.java

```diff
@@ -46,8 +47,14 @@ public int getProductsCount() {
     }
 
     @Step("ProductPage: get product type page visibility status by locating page heading")
-    public Boolean isOpened() {
-        return $x("//h1[@class = 'catalog-heading ng-star-inserted']").is(Condition.visible);
+    public boolean isOpened() {
+        try {
+            return $x("//h1[@class = 'catalog-heading ng-star-inserted']")
+                    .should(Condition.exist)
+                    .is(Condition.visible);
+        } catch (AssertionError exception) {
+            return false;
+        }
```

Comment:
this is duplicated, you can create a separate static method in util class and just re-use it:
return isVisible(element);